packages/browser/src/core/user/index.ts

```diff
@@ -173,6 +190,8 @@ export class LocalStorage extends Store {
   remove(key: string): void {
     return localStorage.removeItem(key)
   }
+
+  getType = (): StoreType => 'localStorage'
```

Comment:
Hey, do we really need this? We can already check instanceOf -- At the very least, I think it would make more sense as a `readonly field` (e.g. readonly type = 'localStorage'), rather than a method?

Reply:
Revising this comment, after delay. The UniversalStorage will be passed into the action-destinations, and I want the action-destinations to have a nice way to choose their storage targets: `i.e., storage.get('test', ['cookies'])`  ... Not sure possible cleanly here 🤔

Reply:
Nice!

Reply:
👍